src/components/Header/userInfo.tsx

```diff
@@ -30,7 +25,16 @@ export const UserInfo: React.SFC<RootStoreProps> = ({rootStore}) => {
             />
           </div>
           <div className="header_login__title">{profileStore.fullName}</div>
-        </div>
+        </DropdownControl>
+        <DropdownContainer>
+          <DropdownList>
+            <DropdownListItem>
+              <a href="javascript:void(0)" onClick={authStore.signOut}>
```

Comment:
rally? `javascript:void(0)` :)

Reply:
Well, i believe that all `a` tags should have `href` attribute and `javascript:void(0)` is safer than `#`) But i'm open for the better approach